rpcs3/Crypto/unpkg.h

```diff
@@ -332,7 +332,7 @@ class package_reader
 	};
 
 public:
-	package_reader(const std::string& path);
+	package_reader(const std::string& path, fs::file file = {});
```

Comment:
why is this needed?

Reply:
As you can see in PR description, it is android stuff
File choose on new android versions returns URI without real fs path, only android knows how to correctly open this shit